examples/speaker_tasks/recognition/README.md

```diff
@@ -9,12 +9,18 @@ Documentation section for speaker related tasks can be found at:
 |:------------------------------:|:---------------------:|:--------------------------------------:|
 | [speakerverification_speakernet](https://ngc.nvidia.com/catalog/models/nvidia:nemo:speakerverification_speakernet) |        xvector        |                  1.96                  |
 |           [ecapa_tdnn](https://ngc.nvidia.com/catalog/models/nvidia:nemo:ecapa_tdnn)           | channel-<br>attention |                  0.92                  |
+|           [titanet_large](https://ngc.nvidia.com/catalog/models/nvidia:nemo:ecapa_tdnn)           | channel-<br>attention |                  0.69                  |
 
 ## Training
 Speaker Recognition models can be trained in a similar way as other models in NeMo using train and dev manifest files. Steps on how to create manifest files for voxceleb are provided below.
-We provide two model configurations based on xvector and modified ecapa_tdnn, with pretrained models provided for each of them. 
+We provide three model configurations based on titanet, speakernet and modified ecapa_tdnn, with pretrained models provided for each of them. 
```

Comment:
Minor Issue: It would be better off if you can make some of the latters uppercase just as SpeakerNet, ECAPA_TDNN in the NeMo docs or the main README.

Reply:
updated in README, docs need to updated, which shall be done as part of docs PR